packages/strapi-plugin-users-permissions/config/queries/bookshelf.js

```diff
@@ -28,10 +28,54 @@ module.exports = {
           qb.orderBy(params.sort);
         }
       }
+
+      if (params.relations) {
+        Object.keys(params.relations).forEach(
```

Comment:
This is exactly that kind of method that should be located in the `strapi-bookshelf/lib/relations.js` file to avoid repetition inside the query files and generated services. 

See https://github.com/strapi/strapi/blob/master/packages/strapi-hook-bookshelf/lib/relations.js